kolibri/auth/api.py

```diff
@@ -113,7 +113,7 @@ def list(self, request):
         elif type(logged_in_user) is AnonymousUser:
             return Response(Facility.objects.all().values_list('id', flat=True))
         else:
-            return Response(logged_in_user.facility_id)
+            return Response([logged_in_user.facility_id])
```

Comment:
> I don't think this is true. A facility user (admin, coach, learner) can only be associated with one facility at a time. Hence only 1 id will be returned.

That is correct. There is a foreign key from FacilityUser to Facility.

(A FacilityUser *can* however be associated with multiple Classrooms or LearnerGroups, through Roles, though we're not exposing that in the MVP)